code/game/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -407,32 +419,38 @@
 	if (!organ || organ.is_dislocated() || organ.dislocated == -1)
 		return FALSE
 
-	user.visible_message("<span class='warning'>[user] begins to dislocate [src]'s [organ.joint]!</span>")
+	user.visible_message("<span class='danger'>[user] begins to dislocate [src]'s [organ.joint]!</span>", "<span class = 'danger'>You begin to dislocate [src]'s [organ.joint]!</span>")
 	if (do_after(user, 100, progress = FALSE))
 		organ.dislocate(1)
-		visible_message("<span class='danger'>[src]'s [organ.joint] [pick("gives way","caves in","crumbles","collapses")]!</span>")
+		user.visible_message("<span class='danger'>[src]'s [organ.joint] [pick("gives way","caves in","crumbles","collapses","dislocates")]!</span>", "<span class = 'danger'>You [pick("cave in","crumble","collapse","dislocate")] [src]'s [organ.joint]!</span>")
 		return TRUE
 	return FALSE
 
 //Breaks all grips and pulls that the mob currently has.
 /mob/living/human/proc/break_all_grabs(mob/living/human/user)
 	var/success = FALSE
 	if (pulling)
-		visible_message("<span class='danger'>[user] has broken [src]'s grip on [pulling]!</span>")
-		success = TRUE
-		stop_pulling()
+		if (pulling != user)
```

Comment:
No, this checks if the person is attacking themselves instead of [src], it's meant to be from different views of the clients to make it more immersive.

Reply:
This checks if the user is not the one being pulled, if it is then we dispense a different visible_message.